main.py

```diff
@@ -109,7 +109,7 @@ def mouse_drag_events(self, event):
             # If the user is dragging the mouse with the right mouse button
             if event.buttons[2] == 1: pass
 
-
+# test
```

Comment:
why did you add a comment here

P.S. 😘

Reply:
gusto ko lang